CMakeLists.txt

```diff
@@ -79,7 +82,11 @@ include(CheckSymbolExists)
 check_symbol_exists(timerfd_create sys/timerfd.h HAVE_TIMERFD)
 check_symbol_exists(getopt_long getopt.h HAVE_GETOPT)
 if(CMAKE_BUILD_TYPE STREQUAL "Release" OR CMAKE_BUILD_TYPE STREQUAL "RelWithDebInfo")
-	add_compile_options(-flto -march=native -Ofast)
+	add_compile_options(-march=native -Ofast)
```

Comment:
I wouldn't do -march=native by default because binaries will not be portable. Maybe we can move that also to a flag or combine it with LTO?